src/OpenTelemetry/Logs/OpenTelemetryLogger.cs

```diff
@@ -47,13 +47,22 @@ public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Except
             {
                 var options = this.provider.Options;
 
+                string formattedMessage = null;
+                if (options.IncludeFormattedMessage)
+                {
+                    if (formatter != null)
+                    {
+                        formattedMessage = formatter(state, exception);
+                    }
+                }
+
                 var record = new LogRecord(
                     options.IncludeScopes ? this.ScopeProvider : null,
                     DateTime.UtcNow,
                     this.categoryName,
                     logLevel,
                     eventId,
-                    options.IncludeFormattedMessage ? formatter(state, exception) : null,
```

Comment:
Could probably just do:

`options.IncludeFormattedMessage ? formatter?.Invoke(state, exception) : null`

Reply:
:D yes of course! Will modify.